client/blocks/importer/wordpress/import-everything/pre-migration/test/index.test.tsx

```diff
@@ -112,9 +136,11 @@ describe( 'PreMigration', () => {
 			onContentOnlyClick,
 		} );
 
-		expect( screen.getByText( 'Take your site to the next level' ) ).toBeInTheDocument();
-		expect( screen.getByText( 'Upgrade and migrate' ) ).toBeInTheDocument();
-		expect( screen.getByText( 'free content-only import option' ) ).toBeInTheDocument();
+		await waitFor( () => {
```

Comment:
Similar to other tests that were fixed, it now needs to wait because of the request (sticker creation) that happens before loading the component.